test/lib/linter/rules/snapshots/CSPCompliance.ts.md

```diff
@@ -79,3 +79,79 @@ Generated by [AVA](https://avajs.dev).
 > Snapshot 1
 
     []
+
+## General: CSPCompliance/NoInlineJS.html
```

Comment:
Why is there new snapshot data for unrelated checks?

Reply:
Fixed it